xwayland/xwayland.c

```diff
@@ -123,9 +123,23 @@ static void exec_xwayland(struct wlr_xwayland *wlr_xwayland) {
 		wlr_xwayland->wl_fd[1], wlr_xwayland->display, wlr_xwayland->x_fd[0],
 		wlr_xwayland->x_fd[1], wlr_xwayland->wm_fd[1]);
 
-	// TODO: close stdout/err depending on log level
+	// Closes stdout/stderr depending on log verbosity
+	enum wlr_log_importance verbosity = wlr_log_get_verbosity();
+	int devnull = open("/dev/null", O_WRONLY | O_CREAT, 0666);
+	if (devnull < 0) {
+		wlr_log_errno(WLR_ERROR, "XWayland: failed to open /dev/null");
+		return -1;
```

Comment:
I'm not sure how to properly exit with an error here, since the function type is void. Simply a `return` or `_exit(EXIT_FAILURE)` (which seemed too harsh for something this nonessential)?

Reply:
`_exit(EXIT_FAILURE)` is fine. If we can't open `/dev/null` something really bad is going on anyway.